includes/class-wc-stripe-webhook-handler.php

```
@@ -1391,6 +1391,86 @@ public function process_account_updated( $notification ) {
+
+		// Set the order being processed for the `wc_stripe_webhook_received` action later.
+		$this->resolved_order = $order;
+
+		$order_helper = WC_Stripe_Order_Helper::get_instance();
+
+		// Lock the order
+		if ( $order_helper->lock_order_payment( $order ) ) {
+			return;
+		}
+
+		$intent_id = $checkout_session->payment_intent;
```

PHPMD (2.15.0)

[error] The variable $intent_id is not named in camelCase. (undefined)

(CamelCaseVariableName)